library/core/src/iter/traits/iterator.rs

```diff
@@ -1802,12 +1802,10 @@ pub trait Iterator {
     /// # Examples
     ///
     /// ```
-    /// #![feature(iter_is_partitioned)]
-    ///
     /// assert!("Iterator".chars().is_partitioned(char::is_uppercase));
     /// assert!(!"IntoIterator".chars().is_partitioned(char::is_uppercase));
     /// ```
-    #[unstable(feature = "iter_is_partitioned", reason = "new API", issue = "62544")]
+    #[stable(feature = "iter_is_partitioned", since = "1.49.0")]
```

Comment:
1.49 is already in beta and 1.50 is going to be in beta soon, so this should probably be 1.51.

Reply:
Changed